### PR TITLE
Update sig-scheduling leads.

### PR DESCRIPTION
**Pull request by ahg-g**

### CODE REVIEW

**Comment by k82cn** [comment]
/lgtm
/approve

Email thread: https://groups.google.com/forum/#!topic/kubernetes-sig-scheduling/UDCOvONwTTg

File: sigs.yaml [hunk 1]
```diff
@@ -1743,8 +1743,8 @@ sigs:
   label: scheduling
   leadership:
     chairs:
-    - github: bsalamat
-      name: Bobby (Babak) Salamat
+    - github: ahg-g
```
Comment (nikhita) [inline sigs.yaml]: You can add @bsalamat as an `emeritus_lead` :)

Example:

https://github.com/kubernetes/community/blob/ee6617c5b7d495b1bf9998c52fabe08143cee3c5/sigs.yaml#L564-L567

Comment (ahg-g) [reply sigs.yaml]: done :)

**Pull request merged**